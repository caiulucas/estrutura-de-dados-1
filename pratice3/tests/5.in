15
Beatriz 39
Isabelly 168
Valentina 187
Heitor 140
Matheus 82
Alicia 194
AnaLiz 102
Eloa 107
Clarice 180
Luna 24
Lara 164
Marina 161
Henrique 94
Gael 52
MariaVitoria 138
