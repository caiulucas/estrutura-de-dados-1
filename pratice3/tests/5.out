Time 1
[L] Alicia 194
[P] Marina 161
[E] Henrique 94

Time 2
[L] Valentina 187
[P] Heitor 140
[E] Matheus 82

Time 3
[L] Clarice 180
[P] MariaVitoria 138
[E] Gael 52

Time 4
[L] Isabelly 168
[P] Eloa 107
[E] Beatriz 39

Time 5
[L] Lara 164
[P] AnaLiz 102
[E] Luna 24

